src/rpc/blockchain.cpp

```diff
@@ -902,22 +904,23 @@ UniValue getblockchaininfo(const JSONRPCRequest& request)
     LOCK(cs_main);
 
     const Consensus::Params& consensusParams = Params().GetConsensus();
+    const CBlockIndex* pChainTip = GetChainTip();
```

Comment:
Same as in the other comments, we are already locking `cs_main`, we can directly access `chainActive` here without lock it again.